cmd/temporalite/main.go

```diff
@@ -6,6 +6,8 @@ package main
 
 import (
 	"fmt"
+	temporalconfig "go.temporal.io/server/common/config"
+	"gopkg.in/yaml.v3"
```

Comment:
Nit: running [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) will sort these imports below the stdlib imports. Also I think we can omit the import alias for the config package.